wandb/sdk/interface/interface_shared.py

```diff
@@ -386,6 +389,10 @@ def _communicate_artifact(self, log_artifact: pb.LogArtifactRequest) -> Any:
         rec = self._make_request(log_artifact=log_artifact)
         return self._communicate_async(rec)
 
+    def _download_artifact(self, download_artifact: pb.DownloadArtifactRequest) -> Any:
+        rec = self._make_request(download_artifact=download_artifact)
+        return self._communicate(rec)
```

Comment:
This waits for the response from the background process for 5s and then times out. Set the timeout to `None` here.